src/Summoner/Template.hs

```diff
@@ -14,23 +14,23 @@ import Data.List (delete)
 import NeatInterpolation (text)
 
 import Summoner.Default (defaultGHC, endLine)
-import Summoner.ProjectData (CustomPrelude (..), GhcVer (..), ProjectData (..), baseNopreludeVer,
-                             latestLts, showGhcVer)
+import Summoner.GhcVer (GhcVer (..), baseVer, latestLts, showGhcVer)
+import Summoner.ProjectData (CustomPrelude (..), ProjectData (..))
 import Summoner.Text (intercalateMap, packageToModule)
 import Summoner.Tree (TreeFs (..))
 
 import qualified Data.Text as T
 
 ----------------------------------------------------------------------------
--- Stack File Creation
+-- Project Directory Creation
 ----------------------------------------------------------------------------
 
 memptyIfFalse :: Monoid m => Bool -> m -> m
 memptyIfFalse p val = if p then val else mempty
 
 -- | Creating template file to use in `stack new` command
```

Comment:
We are not using `stack new` anymore

Reply:
Did we use it before?

Reply:
When it was `hs-init` a long time ago. But then we realised that it's very limited to what we actually need, and it was overcomplicated some stuff.